samples/simple/metrics.js

```diff
@@ -2,7 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const _INSTANCE = new Glean.Glean();
```

Comment:
I know this is not beautiful, I am sure when this is not a POC we can think of something nicer :D